docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,82 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
 
 const headerStyle = {
   whiteSpace: 'pre-line',
 }
 
+interface DefaultBehaviorInfo {
+  name: string
+  url: string
+}
+
 class ComponentDocTag extends React.Component<any, any> {
   getTagDescription = (forTag, fromInfo) => {
     const tags = (fromInfo.docblock && fromInfo.docblock.tags) || []
     return _.result(_.find(tags, 'title', forTag), 'description')
   }
 
+  findDefaultBehaviorInfo = (fetchDefaultBehaviorInfo): DefaultBehaviorInfo => {
+    const accessibilityProperty = fetchDefaultBehaviorInfo.props.find(
+      property => property.name === 'accessibility',
+    )
+    if (!accessibilityProperty) {
+      return undefined
+    }
+
+    const accPropertyFormatted = accessibilityProperty.defaultValue.split('.').pop() + '.ts'
+
+    for (const behavior of behaviorMenuItems) {
+      const variation = behavior.variations.find(v => v.name === accPropertyFormatted)
+      if (variation) {
+        return this.createDefaultBehaviorInfo(behavior, variation)
+      }
+    }
+
+    return undefined
+  }
+
+  createDefaultBehaviorInfo(behavior: any, variation: any): DefaultBehaviorInfo {
+    const behaviorsFolder = behavior.type + 's'
+    const defaultBehaviorFile = variation.name.replace('.ts', '')
+    return {
+      name: `${defaultBehaviorFile}`,
+      url: `${behaviorsFolder}/${behavior.displayName}#${_.kebabCase(defaultBehaviorFile)}`,
+    }
+  }
+
+  renderDefaultBehavior(info) {
```

Comment:
maybe it would be better to make the name of the arg more specific, `info` -> `componentInfo`